sdks/opik_optimizer/scripts/litellm_gepa_hotpot_example.py

```diff
@@ -0,0 +1,84 @@
+"""
+GEPA example on a Hotpot-style dataset using LiteLLM models.
+
+Notes:
+- Ensures local `src/` is imported so the new GEPA optimizer is used.
+- Makes `dspy` optional; falls back to a no-op search tool if not installed.
+"""
+
+from typing import Any, Dict
+
+from opik_optimizer import ChatPrompt
+from opik_optimizer.gepa_optimizer import GepaOptimizer
+from opik_optimizer.datasets import hotpot_300
+
+from opik.evaluation.metrics import LevenshteinRatio
+from opik.evaluation.metrics.score_result import ScoreResult
+
+# Optional tool used by the prompt (same style as other examples)
+import dspy
+
+
+def search_wikipedia(query: str) -> list[str]:
+    results = dspy.ColBERTv2(url="http://20.102.90.50:2017/wiki17_abstracts")(
+        query, k=3
+    )
+    return [item["text"] for item in results]
+
+
+dataset = hotpot_300()
+
+
+def levenshtein_ratio(dataset_item: Dict[str, Any], llm_output: str) -> ScoreResult:
+    metric = LevenshteinRatio()
+    return metric.score(reference=dataset_item["answer"], output=llm_output)
+
+
+system_prompt = "Answer the question"
+
+prompt = ChatPrompt(
+    system=system_prompt,
+    user="{question}",
+    tools=[
+        {
+            "type": "function",
+            "function": {
+                "name": "search_wikipedia",
+                "description": "This function is used to search wikipedia abstracts.",
+                "parameters": {
+                    "type": "object",
+                    "properties": {
+                        "query": {
+                            "type": "string",
+                            "description": "The query parameter is the term or phrase to search for.",
+                        },
+                    },
+                    "required": ["query"],
+                },
+            },
+        },
+    ],
+    function_map={"search_wikipedia": search_wikipedia},
```

Comment:
Don't forget to track the function:

```suggestion
    function_map={"search_wikipedia": opik.track(type="tool")(search_wikipedia)},
```